ffmpeg/nvidia_test.go

```diff
@@ -23,15 +24,16 @@ func TestNvidia_BadCodecs(t *testing.T) {
 	run, dir := setupTest(t)
 	defer os.RemoveAll(dir)
 
-	fname := dir + "/mpeg2.ts"
-	oname := dir + "/out.ts"
+	fname := dir + "/test.flv"
+	//oname := dir + "/out.ts"
```

Comment:
Nit: Remove commented-out code

Reply:
Thanks for the spot, fixed and cleaned up in 9e51cee9fd2d1e6d14169784ed898a63d0caa960